core/src/test/scala/unit/kafka/log/AbstractLogCleanerIntegrationTest.scala

```diff
@@ -42,7 +42,7 @@ abstract class AbstractLogCleanerIntegrationTest {
   private val defaultMinCleanableDirtyRatio = 0.0F
   private val defaultCompactionLag = 0L
   private val defaultDeleteDelay = 1000
-  private val defaultSegmentSize = 256
+  private val defaultSegmentSize = 5120
```

Comment:
Maybe we can make this 2048.

Reply:
Why would we make it less if this works?

Reply:
No strong reason, but the roll logic is an important part of log cleaner integration tests so I didn't want to go too far in the other direction.